site/src/pages/RoadmapPage/transfers/APExamsSection.tsx

```diff
@@ -138,7 +146,7 @@ const APExamsSection: FC = () => {
         Enter the names of AP Exams that you&rsquo;ve taken to clear course prerequisites.
       </SectionDescription>
       {userAPExams.map((exam) => (
-        <APCreditMenuTile key={exam.examName} userExamInfo={exam} />
+        <APCreditMenuTile key={exam.examName} userExamInfo={exam} unread={unreadExams.includes(exam.examName)} />
```

Comment:
hmm I'm not too sure this is the right approach. I was thinking that maybe you could add an `unread?: true` (or `unread?: boolean`) property to each of the items in the course/ap/other transfers rather than maintaining separate lists to do lookups.

Your approach to adding them would also be simpler, as when you click import, you simply need to categorize and `.forEach(t => { t.unread = true; })` (or you could take in `markUnread` as a parameter in the categorize function)

Removing unread indicates would then constitute simply looping through course, ap, and other transfers, and doing `delete t.unread` for each item in the 3 lists.